tests/integration/test_ethereum_tester.py

```diff
@@ -505,6 +492,67 @@ def test_eth_estimate_gas_revert_without_msg(
             )
             w3.eth.estimate_gas(txn_params)
 
+    def test_eth_call_custom_error_revert_with_msg(
+        self,
+        w3,
+        revert_contract,
+        unlocked_account,
+    ) -> None:
+        txn_params = revert_contract._prepare_transaction(
+            fn_name="customErrorWithMessage",
+            transaction={
+                "from": unlocked_account,
+                "to": revert_contract.address,
+            },
+        )
+        with pytest.raises(TransactionFailed, match="execution reverted"):
```

Comment:
We should check the messages on the "with message" cases, right?

Reply:
Yeah, that's a good idea. Updated!